cluster/azure/util.sh

```diff
@@ -239,10 +235,35 @@ function kube-up {
 function kube-down {
     verify-prereqs
 
-    echo "Bringing down cluster"
-    echo
-    echo "You must do this manually (for now)!"
-    echo "This can be done with:"
-    echo "   azure group delete ${AZ_RESOURCE_GROUP}"
+    # required
+    if [[ -z "${AZURE_SUBSCRIPTION_ID:-}" ]]; then
+        echo "AZURE_SUBSCRIPTION_ID must be set"
+        exit 1
+    fi
+    if [[ -z "${AZURE_DEPLOY_ID:-}" ]]; then
+        echo "AZURE_DEPLOY_ID must be set. This selects the deployment (and resource group) to delete."
+        return -1
+    fi
+
+    #optional
+    declare -a destroy_params
+    declare -a docker_params
+    if [[ ${AZURE_DOWN_SKIP_CONFIRM:-} == 1 ]]; then
```

Comment:
In other scripts we use `== "true"` instead of `== 1` but I think either way should be ok as long as it is consistent within the cloud provider.